### DISCUSSION TITLE
Is dropwizard-guicey compatible with Java 21?

### DISCUSSION THREAD

**Discussion by shinlang** [xvik/dropwizard-guicey#381] [Q&A] [answered]
Hi,

does anyone have experience with running dropwizard-guicey on Java 21? Is it possible at all and if not, are there plans to support it in the near future?

Thanks for your help!

**Comment by xvik** [comment, marked as answer]
Should work: I just tried to run all tests with java 21 and it works.
In general, there should be no problems with new java versions (guicey does not use anything non-standard), so if dropwizard and guice are compatible with new java version then guicey should be too.
